src/course-updates/data/thunk.js

```diff
@@ -0,0 +1,111 @@
+import { NOTIFICATION_MESSAGES } from '../../constants';
+import { RequestStatus } from '../../data/constants';
+import { hideProcessingNotification, showProcessingNotification } from '../../generic/processing-notification/data/slice';
+import {
+  getCourseUpdates,
+  getCourseHandouts,
+  createUpdate,
+  editUpdate,
+  deleteUpdate,
+  editHandouts,
+} from './api';
+import {
+  fetchCourseUpdatesSuccess,
+  createCourseUpdate,
+  editCourseUpdate,
+  deleteCourseUpdate,
+  fetchCourseHandoutsSuccess,
+  editCourseHandouts,
+  updateLoadingStatuses,
+  updateSavingStatuses,
+} from './slice';
+
+export function fetchCourseUpdatesQuery(courseId) {
+  return async (dispatch) => {
+    try {
+      dispatch(updateLoadingStatuses({ fetchCourseHandoutsQuery: RequestStatus.IN_PROGRESS }));
+      const courseUpdates = await getCourseUpdates(courseId);
+      dispatch(fetchCourseUpdatesSuccess(courseUpdates));
+      dispatch(updateLoadingStatuses({ fetchCourseHandoutsQuery: RequestStatus.SUCCESSFUL }));
+    } catch (error) {
+      dispatch(updateLoadingStatuses({ fetchCourseHandoutsQuery: RequestStatus.FAILED }));
+    }
+  };
+}
+
+export function createCourseUpdateQuery(courseId, data) {
+  return async (dispatch) => {
+    try {
+      dispatch(updateSavingStatuses({ createCourseUpdateQuery: RequestStatus.PENDING }));
+      dispatch(showProcessingNotification(NOTIFICATION_MESSAGES.saving));
+      const courseUpdate = await createUpdate(courseId, data);
+      dispatch(createCourseUpdate(courseUpdate));
```

Comment:
On edit, the api call is working as expected, but the data does not include the updates made to update. This is probably connected to the change that you made in the wysiwyg editor.

Reply:
Fixed